### PR TITLE
Update building.md

### PR DESCRIPTION
**Pull request by Cpasjuste**
Ubuntu (20.04.2) build fail without this package:

`<pre>CMake Error at /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake:117 (find_package):
  Could not find a package configuration file provided by
  &quot;boost_program_options&quot; (requested version 1.71.0) with any of the
  following names:

    boost_program_optionsConfig.cmake
    boost_program_options-config.cmake

  Add the installation prefix of &quot;boost_program_options&quot; to CMAKE_PREFIX_PATH
  or set &quot;boost_program_options_DIR&quot; to a directory containing one of the
  above files.  If &quot;boost_program_options&quot; provides a separate development
  package or SDK, be sure it has been installed.
Call Stack (most recent call first):
  /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake:182 (boost_find_component)
  /usr/share/cmake-3.16/Modules/FindBoost.cmake:443 (find_package)
  external/psvpfstools/CMakeLists.txt:8 (FIND_PACKAGE)

-- Configuring incomplete, errors occurred!
</pre>`

### CODE REVIEW

**Review by sunho — APPROVED** [review]
Technically, it should use the boost program_option inside externals/boost. But, I think there's nothing to lose by taking this.

File: building.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ cd Vita3K
 - Install dependencies.
 
 ```sh
-sudo apt install git cmake ninja-build libsdl2-dev pkg-config libgtk-3-dev clang
+sudo apt install git cmake ninja-build libsdl2-dev pkg-config libgtk-3-dev clang libboost-program-options-dev
```
Comment (scribam) [inline building.md:74]:
```suggestion
sudo apt install git cmake ninja-build libsdl2-dev pkg-config libgtk-3-dev clang libboost-filesystem-dev libboost-program-options-dev libboost-system-dev
```
https://github.com/Vita3K/Vita3K/blob/78bd06500a486a4aff0e2dc2f16a49d9e5fe740e/.github/workflows/c-cpp.yml#L58

**Comment by pent0** [comment]
If there's no further response on this PR, we will merge this commit modified with your name credit in the PR, and close the PR.

**Review by scribam — APPROVED** [review]
With #1406, building with the embed boost submodule should be easier. In case it doesn't work, there is a mention at the end of the building.md file about using boost system packages. I updated this list to only include the required boost packages.

**Pull request merged**